rsmtool/notebooks/intermediate_file_paths.ipynb

```diff
@@ -11,7 +11,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Click on the hyperlinks below to see the intermediate experiment files generated as part of this experiment."
+    "The following intermediate files were generated as part of this experiment. To examine a file, click on the link. Note that the descriptions provided here are only brief summaries. For more detailed descriptions, please refer to the [RSMTool documentation](https://rsmtool.readthedocs.io/)."
```

Comment:
Link to the intermediate files page? https://rsmtool.readthedocs.io/en/v11.2.0/usage_rsmtool.html#intermediate-files-rsmtool

Reply:
Yeah I was going to but it's not the same for rsmeval so I wasn't sure.

Reply:
can you create the link dynamically with the context?

Reply:
I think the problem is that we re-assign the "context" for `rsmeval` to be `rsmtool` since the section descriptions are the same but I think I have a way around it. Let me work on it.